libraries/AP_Scripting/generator/src/main.c

```diff
@@ -820,11 +920,19 @@ void handle_singleton(void) {
   if (next_token()) {
     error(ERROR_HEADER, "Singleton contained an unexpected extra token: %s", state.token);
   }
+
+  if (strlen(state.metadata) != 0) {
+    error(ERROR_METADATA, "None method singletones do not support metadata");
```

Comment:
What's a "None method singletones"

Reply:
The function is called 'handle_singleton', the only one I did meta data in the above if's is 'handle_method'. I agree is not a very useful message.